Makefile

```diff
@@ -73,14 +73,14 @@ build: builddp buildcni
 docker: ## Build docker image
 	@echo "******  Docker Image    ******"
 	@echo
-	docker build -t localhost:5000/afxdp-device-plugin -f images/amd64.dockerfile .
+	docker build -t afxdp-device-plugin -f images/amd64.dockerfile .
```

Comment:
@patrickog11 - can you remind me why we had "localhost:5000" here?
Is it needed? Was it just a workaround on our side?